.github/workflows/lighthouse-check.yml

```diff
@@ -0,0 +1,28 @@
+name: Test Lighthouse Check with Minimum Score Enforcement
+on:
+  push:
+  pull_request:
+    types: [opened, edited]
+
+jobs:
+  lighthouse-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      
+      - name: Run Lighthouse Check
+        uses: foo-software/lighthouse-check-action@v2.0.5
+        id: lighthouseCheck
+        with:
+          accessToken: ${{ secrets.LIGHTHOUSE_CHECK_GITHUB_ACCESS_TOKEN }}
```

Comment:
By default there is a access token available, I think it should be ok to use it `${{ secrets.GITHUB_TOKEN }}`

> GitHub automatically creates a GITHUB_TOKEN secret to use in your workflow. You can use the GITHUB_TOKEN to authenticate in a workflow run.

https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow#about-the-github_token-secret

Reply:
Awesome, I'll update it then 👍

Reply:
@eddiejaoude it seems there is an error when using that token: "message: 'Resource not accessible by integration',". Maybe it doesn't have permission to make a PR comment? Do you know if it has the `public_repo` permission?

Reply:
Ah yes, in a fork it wont work 🤔  but in a branch it will. It looks good, lets merge it - but I appreciate it wont work for other people who fork the project